quint/cli-tests.md

```diff
@@ -260,6 +260,11 @@ fi
 <!-- !test check rareSpells - Syntax/Types & Effects/Unit tests -->
     quint test ../examples/spells/rareSpells.qnt
 
+### OK on test BoundedUint8
+
+<!-- !test check BoundedUint8 - Syntax/Types & Effects/Unit tests -->
+    quint test --main=BoundedUInt8Test ../examples/spells/BoundedUInt.qnt
```

Comment:
Tests should get run in the examples dashboard. Whats the added value of adding an additional test here?